monitor/sth_fetcher_test.go

```diff
@@ -65,6 +72,13 @@ func TestObserveSTH(t *testing.T) {
 	// Make another STH observation
 	m.fetcher.observeSTH()
 
+	// There should be another fetch observation sample
+	fetchObservations = test.CountCounterVecWithLabels(m.fetcher.stats.sthFetchTotal, labels)
+	if fetchObservations != 2 {
```

Comment:
same small nit here RE: combining the var decl/assign and the if condition:
```suggestion
	if fetchObservations := test.CountCounterVecWithLabels(m.fetcher.stats.sthFetchTotal, labels); fetchObservations  != 2 {
```